reference/gdscript.rst

```diff
@@ -516,6 +525,34 @@ expressions and must be assigned on initialization.
     const e = [1, 2, 3, 4][0]  # constant expression: 1
     const f = sin(20)  # sin() can be used in constant expressions
     const g = x + 20  # invalid; this is not a constant expression!
+    
+Enums
+^^^^^
+
+*Note, only available in Godot 2.2 or higher.*
+
+Enums are basically a shorthand for constants, and are pretty useful if you
+want to assign consecutive integers to some constant.
+
+If you pass a name to the enum, it would also put all the values inside a
+constant dictionary of that name.
+
+::
+
+    enum {TILE_BRICK, TILE_FLOOR, TILE_SPIKE, TILE_TELEPORT}
+    # Is the same as:
+    const TILE_BRICK = 0
+    const TILE_FLOOR = 1
+    const TILE_SPIKE = 2
+    const TILE_TELEPORT = 3
+
+    enum State {STATE_IDLE, STATE_JUMP = 5, STATE_SHOOT}
+    # Is the same as:
+    const STATE_IDLE = 0
+    const STATE_JUMP = 5
+    const STATE_SHOOT = 6
+    const State = {STATE_IDLE = 0, STATE_JUMP = 5, STATE_SHOOT = 6}
```

Comment:
This is a little confusing , I think. I assume the `const State` line is not meant to be bunched up with the `const STATE_` statements, so perhaps you're missing a `# Is also the same as:` type line?